src/DIRAC/FrameworkSystem/Agent/ComponentSupervisionAgent.py

```diff
@@ -531,12 +533,14 @@ def checkURLs(self):
     gConfig.forceRefresh(fromMaster=True)
 
     # get port used for https based services
-    tornadoPortConfigPath = PathFinder.getComponentSection(
-        componentName=os.path.join('/Systems/Tornado'),
+    tornadoSystemInstance = PathFinder.getSystemInstance(
+        system="Tornado",
         setup=self.setup,
-        componentCategory='Port'
     )
-    self._tornadoPort = gConfig.getValue(tornadoPortConfigPath, self._tornadoPort)
+    self._tornadoPort = gConfig.getValue(
+        Path.cfgPath(tornadoSystemInstance, "Port"),
```

Comment:
getSystemInstance just returns, e.g. Certification, right?
So this needs?
```suggestion
        Path.cfgPath('/System/Tornado/', tornadoSystemInstance, "Port"),
```

Reply:
Yes, correct.